databricks/koalas/window.py

```diff
@@ -49,8 +52,20 @@ def _apply_as_series_or_frame(self, func):
             "to handle the index and columns of output.")
 
     def count(self):
+        # Rolling and Expanding have different behavior for `count` in pandas 1.0.0
+        # for now, we consider both pandas < 1.0.0 and pandas >= 1.0.0
```

Comment:
So it was decided to match to pandas' 1.0.0 right?

Reply:
correct, now we can remove this line.

Thanks !